ksql-engine/src/main/java/io/confluent/ksql/analyzer/Analyzer.java

```diff
@@ -261,6 +261,7 @@ private Format getValueFormat(final Sink sink) {
       if (sink.getProperties().getValueDelimiter().isPresent()) {
         return sink.getProperties().getValueDelimiter();
       }
+
```

Comment:
extra new line?

Reply:
it's intentional - I thought it was cleaner to read with a new line than without